README.md

```diff
@@ -1,13 +1,16 @@
 # Survey form with React
 
-Replace this readme with your own information about your project.
+Survey with at least 3 questions and one summary built with React using states.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Did codealong with the Technigo lessons.
```

Comment:
Setting up the structure of your work is a good idea so you don't get confused about doing different thing all at once!